libraries/SRV_Channel/SRV_Channel_aux.cpp

```diff
@@ -46,7 +46,13 @@ void SRV_Channel::output_ch(void)
             if (SRV_Channels::passthrough_disabled()) {
                 output_pwm = c->get_radio_trim();
             } else {
-                output_pwm = c->get_radio_in();
+                // if the input value has not been changed, do nothing
+                uint16_t radio_in = c->get_radio_in();
+                if (previous_pwm[passthrough_from] < radio_in - c->get_dead_zone() ||
+                        radio_in + c->get_dead_zone() < previous_pwm[passthrough_from]) {
+                    output_pwm = radio_in;
+                    previous_pwm[passthrough_from] = radio_in;
```

Comment:
we need a flag to say if a do_set_servo has been used. If it hasn't then we need to avoid jumps in the output (the change effectively adds a granularity in output)

Reply:
I added the flag  specifying that DO_SET_SERVO has been used.
the output_pwm is always used if use_do_set_servo is false.
It is used only when the value of the transmitter changes if use_do_set_servo is true.